### PR TITLE
Test data module

### PR DESCRIPTION
**Pull request by javicid**

### CODE REVIEW

**Comment by javicid** [comment]
This addresses #27

File: dislib/data/classes.py [hunk 1]
```diff
@@ -81,7 +84,9 @@ def concatenate(self, subset, remove_duplicates=False):
         if remove_duplicates:
             self._ids, uniques = np.unique(self._ids, return_index=True)
             self.samples = self.samples[uniques]
-            self.labels = self.labels[uniques]
+
+            if self.labels is not None:
```
Comment (kafkasl) [inline dislib/data/classes.py]: isn't this the same as `if self.labels:` ?

Comment (javicid) [reply dislib/data/classes.py]: No, self.labels is a numpy array, so 'if self.labels' returns an array of booleans

Comment (kafkasl) [reply dislib/data/classes.py]: :+1:

**Pull request merged**